configure.in

```diff
@@ -518,7 +520,23 @@ AC_CHECK_DECL([PACKET_FANOUT],
         [[#include <linux/if_packet.h>]])
 
 # If we use DPDK we might be able to use libnuma
-AC_CHECK_LIB(numa, numa_node_to_cpus, have_numa=1, have_numa=0)
+AC_ARG_WITH(numa,
+	AS_HELP_STRING(--with-numa,include NUMA support),
+[
+	if test "$withval" = no
+	then
+		want_numa=no
+	else
+		want_numa=yes
+	fi
+],[
+	# Default to building without NUMA
```

Comment:
It looks like "without" should actually be "with" in this comment, correct?

Reply:
You're right, yes! There's another inconsistency too: we should say --without-numa if NUMA is the default (just a convention thing).

I'll fix the comment and the description (to --without). Both --with and --without will continue to work, it's just about what's shown in `./configure --help`.